code/game/objects/items/handcuffs.dm

```diff
@@ -98,6 +99,21 @@
 	cuffsound = 'sound/weapons/cablecuff.ogg'
 	var/datum/robot_energy_storage/wirestorage = null
 
+/obj/item/restraints/handcuffs/cable/Initialize(mapload, color_override)
+	. = ..()
+
+	var/list/cable_colors = GLOB.cable_colors
+	if(color_override)

```

Comment:
```
item_color = color_override || item_color || pick(cable_colors)
if(cable_colors[cable_color])
	cable_color = cable_colors[cable_color]
```

Choose to use either color_override, override_color or param_color and don't use other ones.